R/cancensus.R

```diff
@@ -148,8 +148,10 @@ get_census <- function (dataset, level, regions, vectors=c(), geo_format = "sf",
     result <- if (is.null(result)) {
       geos
     } else if (geo_format == "sf") {
-      dplyr::select(result, -Population, -Dwellings, -Households, -Type) %>%
-        dplyr::inner_join(geos, by = "GeoUID")
```

Comment:
You could use `dplyr::inner_join(geos, ., by = "GeoUID") instead, if you'd like.

Reply:
Thanks. Updated.